<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibration</class>
 <widget class="QWidget" name="calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Modelabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>System Calibration</string>
   </property>
  </widget>
  <widget class="QLabel" name="Power_Display_data_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>default</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Power_Display_Title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>151</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Power : </string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Frequency_Display_data_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>default</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Frequency_Display_Title_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>211</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frequency   :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Display_Title_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>161</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Setting_Title_label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>171</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>20</width>
     <height>371</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="InternalTemp_Display_data_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>default</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InternalTemp_Display_title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>251</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Internal Temp :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Frequency_Setting_Title_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>221</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frequency   :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Power_Setting_Title_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>151</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Power : </string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="Power_line">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>300</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Power_Setting_lcd">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="Line" name="Frequency_line">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>400</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Frequency_Setting_lcd">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>340</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="down_button_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="up_button_label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>440</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_button">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>60</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="AMP_Connect_State_Color_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AMP_Connect_State_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>531</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AMP Connect :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
